From: Scott Kitterman <scott@kitterman.com>
Date: Mon, 13 Mar 2023 00:22:12 -0400
Subject: Add DEBCI to CI environments in tests

---
 tests/test_all.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/test_all.py b/tests/test_all.py
index dfb680e..12fd8a7 100755
--- a/tests/test_all.py
+++ b/tests/test_all.py
@@ -16,7 +16,7 @@ class DNSTest(unittest.TestCase):
 
     def setUp(self):
         self.channel = pycares.Channel(timeout=10.0, tries=1, servers=['8.8.8.8', '8.8.4.4'])
-        self.is_ci = os.environ.get('APPVEYOR') or os.environ.get('TRAVIS') or os.environ.get('GITHUB_ACTION')
+        self.is_ci = os.environ.get('APPVEYOR') or os.environ.get('TRAVIS') or os.environ.get('GITHUB_ACTION') or os.environ.get('DEBCI')
 
     def tearDown(self):
         self.channel = None
@@ -37,7 +37,7 @@ class DNSTest(unittest.TestCase):
                 self.channel.process_fd(pycares.ARES_SOCKET_BAD, fd)
 
     def assertNoError(self, errorno):
-        if errorno == pycares.errno.ARES_ETIMEOUT and self.is_ci:
+        if (errorno == pycares.errno.ARES_ETIMEOUT or errorno == pycares.errno.ARES_ESERVICE) and self.is_ci:
             raise unittest.SkipTest('timeout')
         self.assertEqual(errorno, None)
 
